mrp_workcenter_workorder_link/mrp_view.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+<!-- mrp.workcenter -->
+
+<record id="mrp_workcenter_tree_view" model="ir.ui.view">
+    <field name="model">mrp.workcenter</field>
+    <field name="inherit_id" ref="mrp.mrp_workcenter_tree_view"/>
+    <field name="arch" type="xml">
+        <field name="name" position="before">
+            <button name="button_workcenter_line"
+                    string="Click to see workcenter's operations"
+                    icon="gtk-execute"
+                    type="object"/>
```

Comment:
I understand but some other modules will add other elements to this view. Then user don't want to be confused by too much information. This module is just an UI improvement for other higher level modules. In new UI steps for Odoo versions, icons will win, I suppose. 
Please consider this like a transitional step on a not so young version. Thank you